packages/core/content-type-builder/admin/src/components/FormModal/forms/index.js

```diff
@@ -191,12 +191,18 @@ const forms = {
         return get(contentType, ['schema', 'collectionName'], '');
       });
 
-      const takenCollectionNames = isEditing ? collectionNames.filter((collectionName) => {
-        const currentPluralName = get(contentTypes, [ctUid, 'schema', 'pluralName'], '');
-        const currentCollectionName = get(contentTypes, [ctUid, 'schema', 'collectionName'], '');
+      const takenCollectionNames = isEditing
+        ? collectionNames.filter((collectionName) => {
+            const currentPluralName = get(contentTypes, [ctUid, 'schema', 'pluralName'], '');
+            const currentCollectionName = get(
+              contentTypes,
+              [ctUid, 'schema', 'collectionName'],
+              ''
+            );
```

Comment:
We could try and get rid of lodash/get here:

```suggestion
            const { schema } = contentTypes[ctUid];
            const currentPluralName = schema.pluralName;
            const currentCollectionName = schema.collectionName,
              ''
            );
```

I'm not sure, but I don't think we need optional chaining here, because all of these are mandatory schema properties, I think?

Reply:
I've tried to remove all lodash usage from the FormModal. wdyt?

Reply:
I like it already without having a look :p